script.js

```diff
@@ -0,0 +1,43 @@
+/* HW_1
```

Comment:
I see that you wanted to comment “HW_1”, but closed your comment in the line 12, but not in the first line

Reply:
I messed up a bit with branches, that's why it is a bit messy here. Corrected it in the next pull request :)